web/web.go

```diff
@@ -96,7 +96,7 @@ var (
 			Help:    "Histogram of latencies for HTTP requests.",
 			Buckets: []float64{.1, .2, .4, 1, 3, 8, 20, 60, 120},
 		},
-		[]string{"handler"},
+		[]string{"handler", "code"},
```

Comment:
That's getting into high cardinality, and breaking out latency by success/failure can be problematic. I'd suggest a new counter metric.

Reply:
Makes sense, I'll update accordingly.